src/main/scala/org/broadinstitute/dsde/workbench/avram/db/CollectionComponent.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.avram.api.Collection
+import AvramPostgresProfile.api._
+
+import scala.concurrent.ExecutionContext
+
+case class CollectionRecord(id: Long,
+                            name: String,
+                            createdBy: String,
+                            samResource: String,
+                            createdTimestamp: Timestamp,
+                            updatedBy: String,
+                            updatedTimestamp: Timestamp)
+
+trait CollectionComponent extends AvramComponent {
+
+  class CollectionTable(tag: Tag) extends Table[CollectionRecord](tag, "collection") {
+    def id =                     column[Long]            ("id",                  O.PrimaryKey, O.AutoInc)
```

Comment:
Do we have to include the column options like `O.PrimaryKey`? According to the slick docs (http://slick.lightbend.com/doc/3.2.3/schemas.html), it looks like they're only used when creating DDL statements... a feature of slick we're not using because we're using liquibase instead. I think we can safely omit all of the slick column options.

Reply:
I kind of like having column options, keys and indexes here so I can see the structure of the table - but I don't really have a strong opinion either way.

Reply:
Assuming they turn out to be completely non-functional, my strongest argument against having them is that one might think they can change behavior by changing the flags here.

Reply:
I didn't realize they were just used for DDL updates (but that makes sense). I kind of like having them -- they are useful as documentation and I worry about some undocumented slick behavior if we omit them. Maybe leave them but include a comment next to the record definitions that we use liquibase for DDL updates instead of slick?

Reply:
If we want documentation, we can add documentation comments. I still feel that code with zero effect is distracting and misleading.

Reply:
I would like to resolve this in a future PR and not here.